apps/emqx_gateway/src/stomp/emqx_stomp_channel.erl

```diff
@@ -292,10 +295,9 @@ ensure_connected(Channel = #channel{
                               ctx = Ctx,
                               conninfo = ConnInfo,
                               clientinfo = ClientInfo}) ->
-    NConnInfo = ConnInfo#{connected_at => erlang:system_time(millisecond)},
```

Comment:
Plz revert this change.
`connected_at` should only be set during connection state transitions, not enrich_conninfo.